src/test/resources/db/scripts/schema.sql

```diff
@@ -42,7 +42,7 @@
         target varchar(255),
         direction enum ('INBOUND','OUTBOUND'),
         message_body longtext,
-        type enum ('EMAIL','SMS'),
+        type enum ('EMAIL','SMS','WEB_MESSAGE'),
```

Comment:
Why not change to varchar instead?
I don't see the point with enums in the database, (given how we are using it). 

It _might_ have been useful, if we had different sources that inserted content in the database. That is not the case here.